app/services/strategy_marketplace_service.py

```
@@ -1176,76 +1176,102 @@ async def _get_ab_test_results(self, strategy_func: str) -> Dict[str, Any]:
             "improvement": 22.7
         }
     
-    async def _get_community_strategies(self, user_id: str) -> List[StrategyMarketplaceItem]:
+    async def _get_community_strategies(
+        self,
+        user_id: str,
```

Ruff (0.13.1)

Unused method argument: `user_id`

(ARG002)